checkstyle.xml

```diff
@@ -81,13 +81,13 @@
   <module name="TreeWalker">
 
     <!-- Checks for Javadoc comments. -->
-    <!-- See http://checkstyle.sf.net/config_javadoc.html -->
+    <!-- See https://checkstyle.org/config_javadoc.html -->
```

Comment:
https://checkstyle.org/config_javadoc.html#JavadocMethod

Reply:
No, this comment is also for MissingJavadocMethod.